file_name_in: "/MFG/disc.csv"
file_name_out: "/MFG/disc"